roles/bahmni-emr/tasks/main.yml

```diff
@@ -7,18 +7,18 @@
   yum: name="openmrs" state=absent
   ignore_errors: true
 
-- name: Check if bahmni-openmrs rpm needs an update
-  command: yum list installed bahmni-openmrs
-  register: installed_openmrs
-  ignore_errors: true
+# - name: Check if bahmni-openmrs rpm needs an update
+  # command: yum list installed bahmni-openmrs
+  # register: installed_openmrs
+  # ignore_errors: true
 
 # Uninstall 2FA app before uninstalling OpenMRS
 - name: Uninstall two factor authentication rpm
   yum:  name="bahmni-two-factor-auth" state=absent
 
-- name: Uninstall bahmni-openmrs rpm
-  yum: name="bahmni-openmrs" state=absent
-  when: installed_openmrs.stdout.find("{{ bahmni_openmrs_version }}") == -1
+# - name: Uninstall bahmni-openmrs rpm
+  # yum: name="bahmni-openmrs" state=absent
+  # when: installed_openmrs.stdout.find("{{ bahmni_openmrs_version }}") == -1
```

Comment:
Why are we commenting instead of removing these lines? @shilpa-iplit

Reply:
Removed the lines in updated PR.